doc/man1/openssl.pod

```diff
@@ -75,7 +75,7 @@ B<list>, or B<no->I<XXX> itself.)
 
 =head2 Standard Commands
 
-=over 10
+=over 4
```

Comment:
This makes it look totally different. 10 happens to be longer than most of the commands, making it come just after it instead of on the next line.